### PR TITLE
Save&restore support for NTTable

### PR DESCRIPTION
**Pull request by georgweiss**

### CODE REVIEW

File: core/pva/src/main/java/org/epics/pva/client/PutRequest.java [hunk 1]
```diff
@@ -226,8 +236,11 @@ else if (subcmd == PVAHeader.CMD_SUB_DESTROY)
             // Indicate completion now that server confirmed PUT
             complete(null);
         }
+        /*
```
Comment (kasemir) [inline core/pva/src/main/java/org/epics/pva/client/PutRequest.java:239]: Do we indeed need to disable the exception?

File: core/pva/src/main/java/org/epics/pva/data/PVAData.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 public abstract class PVAData
 {
     /** Name */
-    protected final String name;
+    protected String name;
```
Comment (kasemir) [inline core/pva/src/main/java/org/epics/pva/data/PVAData.java:25]: Does the name need to change, no longer final?

**Comment by georgweiss** [comment]
Let me check...

**Comment by georgweiss** [comment]
@kasemir , changes reverted. Thanks for pointing out.

**Review by kasemir — APPROVED** [review]

**Pull request merged**